src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier cs = new ColorSupplier();
+    private int maximalSizeOfSide = 100;
+    private int boundNumberOfFigureClasses = 5;
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/14baae8a-a8dc-4f44-b7b8-9d2dcd038238)

You should fix (or discuss in Slack if disagree) all mentor's comments before re-request new review. Be attentive and do not hesitate to contact us in Slack if you do not understand something.